src/dmd/doc.d

```diff
@@ -1653,6 +1653,19 @@ struct DocComment
             p = skipwhitespace(p);
             pstart = p;
             pend = p;
+
+            // Undo indent if starting with a list item
+            if ((*p == '-' || *p == '+' || *p == '*') && (*(p+1) == ' ' || *(p+1) == '\t'))
```

Comment:
* Perhaps this should condition should be its own function
* Perhaps a constant should be defined somewhere containing the possible characters a list can start with